src/Microsoft.FeatureManagement.AppInsightsTelemetryPublisher/Microsoft.FeatureManagement.Telemetry.AppInsights.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
A new package is introduced for the app insights telemetry publisher. Let us all agree on the package name.

Curently the PR has

`Microsoft.FeatureManagement.Telemetry.AppInsights`

But I'm thinking the following may fit better

`Microsoft.FeatureManagement.Telemetry.AppInsights`

cc @rossgrambo @zhenlan @drago-draganov @zhiyuanliang-ms @amerjusupovic

Reply:
Given the app insights package name is Microsoft.App**lication**Insights, I think we should match.

`Microsoft.FeatureManagement.Telemetry.ApplicationInsights`

Reply:
Good point, I'll adjust to that.